tests/components/totalconnect/snapshots/test_binary_sensor.ambr

```diff
@@ -895,6 +1037,102 @@
     'state': 'off',
   })
 # ---
+# name: test_entity_registry[binary_sensor.test_safety-entry]
```

Comment:
Before merging this, all the snapshots need to be deleted and regenerated

Reply:
Done in [54d09df](https://github.com/home-assistant/core/pull/121888/commits/54d09df033e3a82b5a417a10ca3d3b5c947c7c80)